soc/arm/nuvoton/m48x/soc.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2020 Linumiz
+ * Author: Saravanan Sekar <saravanan@linumiz.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _M48X_SOC_H_
```

Comment:
Should use a conflict-free identifier; also should not use identifiers that start with an underscore as per various coding standards.

```suggestion
#ifndef ZEPHYR_SOC_ARM_NUVOTON_M48X_SOC_H_
```

Reply:
ok I will change